test/server/routes/applications.test.js

```diff
@@ -382,6 +389,7 @@ describe('routes/applications.js', () => {
     return request(app)
       .post(slug)
       .set('Cookie', signedInCookie)
+      .send({ _csrf: CSRF_TOKEN })
```

Comment:
so rather than just send the `csrf` token, i believe you're meant to send the entire form data, e.g `routeData.addApplication` which includes the token as well..